### DISCUSSION TITLE
Is the Podman formula built for Linux?

### DISCUSSION THREAD

**Discussion by henryhchchc** [Homebrew/discussions#700] [Linux] [answered]
Seems podman installed via `brew install podman` is not executable on Linux

**Comment by Bo98** [comment, marked as answer]
It is not. It looks like the formula does things like `make podman-remote-darwin`.

If you're familiar with the software, you can open a pull request to add Linux build support to it.

**Reply by henryhchchc** [reply]
Thanks for the confirmation. I am still struggling with the dependency issues. Will open a PR if I get it working.
